shell/packages/sandstorm-db/db.js

```diff
@@ -415,9 +423,17 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //       }
 //     }
 //     frontendRef :union {
-//        notificationHandle :Text;
-//        ipNetwork :Bool;
-//        ipInterface :Bool;
+//       notificationHandle :Text;
+//       ipNetwork :Bool;
+//       ipInterface :Bool;
+//       emailVerifier :group {
+//         services :List(String);
+//       }
+//       verifiedEmail :group {
+//         verifierId :Text;
+//         tabId :Text;
```

Comment:
If I'm reading this correctly, this means that if your app requests a `VerifiedEmail`, and then the user terminates that session and comes back later, they're going to have a different `tabId`, so the VerifiedEmail won't verify if passed to `EmailVerifier` any more.

This basically makes `save()` and `restore()` on `VerifiedEmail` useless for apps.  Is the idea that the app would request `VerifiedEmailSendPort` once, verify the email immediately before saving it durably, and then only ever call `EmailSendPort`'s methods on the capability you received thereafter?